ProjectManagmentSystem/src/Modules/AuthenticationModule/components/Register/Register.tsx

```diff
@@ -1,7 +1,238 @@
-import React from 'react'
+import axios from 'axios';
+import React, { useRef, useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router-dom';
+import logo from "../../../../assets/images/PMS 3.png";
+import Styles from "./Register.module.css"
+
+import img from "../../../..//assets/images/8550fbcbe60cd242d12760784feff287.jpeg";
 
 export default function Register() {
+
+  const [visible, setVisible] = useState(false);
+  
+  const navigate = useNavigate();
+  let {
+    register,
+    handleSubmit,
+    formState: { errors },
+    watch,
+  } = useForm();
+  const imgValue = watch()
+ 
+  
+  
+  const password = useRef({});
+  password.current = watch("password", "");
+  const appendToFormData = (data:any) => {
```

Comment:
We can make this a general utility that will handle any data and then use it when needed instead of having it in every component that contains a form.